packages/app/lib/ui/screens/species.dart

```diff
@@ -93,25 +94,45 @@ class _SpeciesProfileState extends State<SpeciesProfile> {
   @override
   Widget build(BuildContext context) {
     return Container(
-      padding: const EdgeInsets.only(
-          top: 10.0, right: 20.0, bottom: 20.0, left: 20.0),
+      padding: const EdgeInsets.only(top: 0.0, right: 20.0, left: 20.0),
       decoration: const SeaWavesBackground(),
-      child: Wrap(runSpacing: 20.0, children: [
-        SpeciesProfileTitle(species.species.name),
-        const SizedBox(height: 100.0),
-        SpeciesField(
-          species.species,
-          allowNull: false,
-          onUpdate: _updateTaxon,
-        ),
-        EditableTextField(species.description,
-            title: AppLocalizations.of(context)!.speciesDescription,
-            onUpdate: _updateDescription),
-      ]),
+      child: CustomScrollView(
+        slivers: [
+          SliverList(
+              delegate: SliverChildListDelegate([
+            SpeciesProfileTitle(species.species.name),
+            const SizedBox(height: 100.0),
+            SpeciesField(
+              species.species,
+              allowNull: false,
+              onUpdate: _updateTaxon,
+            ),
+            const SizedBox(height: 20.0),
+            EditableTextField(species.description,
+                title: AppLocalizations.of(context)!.speciesDescription,
+                onUpdate: _updateDescription),
+            const SizedBox(height: 20.0),
+          ])),
+          RelatedSightings(id: species.id),
+          const SliverToBoxAdapter(child: SizedBox(height: 20.0)),
```

Comment:
When not using a sliver but a widget we can wrap it around a `SliverToBoxAdapter`